packages/components/src/internal/components/user/UserProfile.tsx

```diff
@@ -203,13 +212,14 @@ export class UserProfile extends PureComponent<Props, State> {
     }
 
     render() {
+        const { userProperties } = this.props;
```

Comment:
I think we should update the Groups listing on this page to also link to the groups, and show the effective roles. That could be done in a separate PR.

Reply:
I updated the profile page to use the GroupsList component for display, but will hold off on showing the Effective Roles for another PR.